x/distribution/keeper/keeper.go

```diff
@@ -38,7 +38,7 @@ func NewKeeper(
 // AllocateTokens modifies the fee distribution by:
 // - Allocating the community tax portion to the community pool
 // - Burning all remaining tokens instead of distributing to validators
-func (k Keeper) AllocateTokens(ctx sdk.Context, _, _ int64, _ sdk.ConsAddress, _ []abci.VoteInfo) {
+func (k Keeper) AllocateTokens(ctx sdk.Context, _ int64, _ []abci.VoteInfo) {
```

Comment:
does this mirror the original distr keeper?

Reply:
Yes, the signature has been changed.